src/js/Features.js

```diff
@@ -165,23 +165,17 @@ Features.prototype.generateElements = function (featuresElements) {
             if (!self._features[i].hideName) {
                 featureName = `<td><div>${self._features[i].name}</div></td>`;
             }
+            const newElement = $(`<tr><td><input class="feature toggle" id="feature-${i}" \
+                " name="${self._features[i].name}" title="${self._features[i].name}" type= \
```

Comment:
I think this `"` starting is not needed, the id is closed in the earlier line.

Reply:
Thanks. Good spot! Refactored and tested.